dot/state/transaction.go

```diff
@@ -68,7 +70,16 @@ func (s *TransactionState) RemoveExtrinsicFromPool(ext types.Extrinsic) {
 // AddToPool adds a transaction to the pool
 func (s *TransactionState) AddToPool(vt *transaction.ValidTransaction) common.Hash {
 	s.notifyStatus(vt.Extrinsic, transaction.Future)
-	return s.pool.Insert(vt)
+
+	hash := s.pool.Insert(vt)
+
+	if err := telemetry.GetInstance().SendMessage(
+		telemetry.NewTxpoolImportTM(uint(s.queue.Len()), uint(s.pool.Len())),
+	); err != nil {
+		logger.Debug("problem sending txpool.import telemetry message", "error", err)
```

Comment:
Merge the dev branch and change to

```suggestion
		logger.Debugf("problem sending txpool.import telemetry message: %s", err)
```

nit also should we change that to a warning instead? :thinking:

Reply:
done